lib/Command/InstallModelsCommand.php

```diff
@@ -0,0 +1,207 @@
+<?php
+/**
+ * @copyright Copyright (c) 2019, Matias De lellis <mati86dl@gmail.com>
+ *
+ * @author Matias De lellis <mati86dl@gmail.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+namespace OCA\FaceRecognition\Command;
+
+use OCP\ITempManager;
+use OCP\App\IAppManager;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class InstallModelsCommand extends Command {
```

Comment:
so, I was wondering... can we make this a bit generic. We already discussed this in the form of `./occ face:recommend_memory`. Can you name this command, someting like `./occ face:setup` or `./occ face:prepare` or `./occ face:first_time_setup` or `./occ face:initialize`.... point is, we can use this same command later, not just for downloading models, but this is a place we can tell people "hey, your recommended memory is not high" or "hey, DLib does not exist", or "hey, PDLib cannot be initialized"... instead of having 2-3-5 commands for this, why not make it one? At this moment, it can contain only models, but in the future, we can bundle all various things in here. First next step to merge in this command would be check requirements

Reply:
OK.. `./occ face:setup`
Now without an argument, but then `./occ face:setup model 2`, `./occ face:setup memory`  can be added.  I did not like `setup`, but it is the most generic, and can be consistent with the arguments.